internal/fwschema/attribute_default.go

```diff
@@ -0,0 +1,77 @@
+package fwschema
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/defaults"
+)
+
+// AttributeWithBoolDefaultValue is an optional interface on Attribute which
+// enables Bool default value support.
+type AttributeWithBoolDefaultValue interface {
+	Attribute
+
+	DefaultValue() defaults.Bool
```

Comment:
If I recall, we made these interface methods `{TYPE}PlanModifiers()` in the attribute plan modification interfaces so that technically an attribute could be created that satisfied all the typed interfaces. Not sure if we want to continue that pattern.

Reply:
Sounds like a good idea. Have renamed functions to `{TYPE}DefaultValue()`.